e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -12,56 +12,63 @@
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 
 describe('System Console > Site Statistics', () => {
-    it('MM-T903 - Site Statistics > Deactivating a user increments the Daily and Monthly Active Users counts down', () => {
-        cy.apiInitSetup().then(({team, user}) => {
-            const testUser = user;
+    let testUser;
+    let totalActiveUsersInitial;
+    let dailyActiveUsersInitial;
+    let monthlyActiveUsersInitial;
+    let totalActiveUsersFinal;
+    let dailyActiveUsersFinal;
+    let monthlyActiveUsersFinal;
```

Comment:
While here, I think the declaration here should only be for `let testUser;`. The rest should be declared on where it's been assigned with a value, ex:
```javascript
const totalActiveUsersInitial = parseInt(totalActiveText, 10);
```

Reply:
I thought about it too.. but also thought that if they are declared above, they could be reused by other site statistics tests that may possibly be added later. But you're right these should be declared where used and can be move above if more tests are added in the future and they use it.